plugins/hue/hue

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/python3 -tt
+# -*- coding: utf-8 -*-
+
+"""Munin plugin to read various data from Philips Hue.
+
+Copyright 2019, Kim B. Heino, b@bbbs.net, Foobar Oy
+License GPLv2+
+
+Munin configuration:
+
+[hue]
+env.host hue-bridge-ip-or-hostname
+env.auth auth-key
+"""
+
+import os
+import sys
+import unicodedata
+import requests
+
+
+def cleanup(value):
+    """Convert value to Munin key."""
+    ret = []
+    for char in value:
+        if 'a' <= char <= 'z' or '0' <= char <= '9':
+            ret.append(char)
+        elif 'A' <= char <= 'Z':
+            ret.append(char.lower())
+        else:
+            ret.append('_')
+    return ''.join(ret)
+
+
+def sensor_name(data, sensor):
+    """Find "parent" sensor for temperatue/light."""
+    uniqueid = sensor['uniqueid'][:27]
+    name = sensor['name']
+    for item in data['sensors'].values():
+        if item.get('uniqueid', '').startswith(uniqueid):
+            name = item['name']
+            break
+
+    # Convert ä to a and return it as utf8 string, py2/3 compatible way
+    name = unicodedata.normalize('NFKD', name)
+    return name.encode('ASCII', 'ignore').decode('utf-8')
```

Comment:
Is it necessary to reformat the name here? Munin should handle special characters well, I think.

Reply:
Munin 2.0.6something on RHEL7 does *not* support UTF-8. It might be because of old rrdtool instead of Munin, but the result looks bad. So I insist doing ä->a conversion here.

Reply:
That sounds reasonable.
But please add a comment referring to RHEL7 (improves understanding and allows retiring of this workaround, when RHEL7 goes end-of-life).
The issue should be fixed since Munin 2.0.11 (bb9466db1813ca9bc50e74e67ecbf49bcd011095).